modules/mobile/app/models/mobile/v0/adapters/lighthouse_individual_claims.rb

```diff
@@ -0,0 +1,187 @@
+
+        # Order of EVENT_DATE_FIELDS determines which date trumps in timeline sorting.
+        EVENT_DATE_FIELDS = %i[
+          closed_date
+          received_date
+          upload_date
+          opened_date
+          requested_date
+          suspense_date
+        ].freeze
+
+        # Lighthouse only provides tracked item status which does not match 1-1 to evss tracked item types.
+        # We're forced here to coerce the status into a type that may not necessarily be true. but will behave
+        # identically in the app. All received statuses are set to received_from_you_list when they could have actually
+        # been received_from_others_list type.
+        LH_STATUS_TO_EVSS_TYPE = {
+          ACCEPTED: 'received_from_you_list',
+          INITIAL_REVIEW_COMPLETE: 'received_from_you_list',
+          NEEDED_FROM_YOU: 'still_need_from_you_list',
+          NEEDED_FROM_OTHERS: 'still_need_from_others_list',
+          NO_LONGER_REQUIRED: 'received_from_you_list',
+          SUBMITTED_AWAITING_REVIEW: 'received_from_you_list'
+        }.freeze
+
+        LH_STATUS_TO_EVSS_STATUS = {
+          ACCEPTED: 'NO_LONGER_REQUIRED',
+          INITIAL_REVIEW_COMPLETE: 'NO_LONGER_REQUIRED',
+          NEEDED_FROM_YOU: 'NEEDED',
+          NEEDED_FROM_OTHERS: 'NEEDED',
+          NO_LONGER_REQUIRED: 'NO_LONGER_REQUIRED',
+          SUBMITTED_AWAITING_REVIEW: 'NEEDED'
+        }.freeze
+
+        UPLOADED_STATUSES = %w[ACCEPTED INITIAL_REVIEW_COMPLETE SUBMITTED_AWAITING_REVIEW].freeze
+        def create_tracked_item_event(tracked_item, tracked_item_documents)
+          documents = create_documents(tracked_item_documents)
+
+          event = {
+            type: LH_STATUS_TO_EVSS_TYPE[tracked_item['status'].to_sym],
+            tracked_item_id: tracked_item['id'],
+            description: tracked_item['description'],
+            display_name: tracked_item['displayName'],
+            overdue: tracked_item['overdue'],
+            status: LH_STATUS_TO_EVSS_STATUS[tracked_item['status'].to_sym],
+            uploaded: UPLOADED_STATUSES.include?(tracked_item['status']),
+            uploads_allowed: tracked_item['uploadsAllowed'],
+            opened_date: tracked_item['requestedDate'],
+            requested_date: tracked_item['requestedDate'],
+            received_date: tracked_item['receivedDate'],
+            closed_date: tracked_item['closedDate'],
+            suspense_date: tracked_item['suspenseDate'],
+            documents:,
+            upload_date: latest_upload_date(documents)
+          }
+          event[:date] = Date.strptime(event.slice(*EVENT_DATE_FIELDS).values.compact.first, '%Y-%m-%d')
+          event
+        end
+
+        def create_documents(documents)
+          documents.map do |document|
+            {
+              tracked_item_id: document['trackedItemId'],
+              file_type: document['documentTypeLabel'],
+              # no document type field available
+              document_type: nil,
```

Comment:
document_type does not have a corresponding field in the LH version. It is not used in FE though.